### PR TITLE
Tool tip

### PR DESCRIPTION
**Pull request by shuhaib-aot**
# Issue Tracking

JIRA: 

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

**Review by kurianbenoy-aot — CHANGES_REQUESTED** [review]
Issue in code when running in console due to function Error. Fix the issue

File: camunda-formio-tasklist-vue/src/services/FormatTime.ts [hunk 1]
```diff
@@ -6,15 +6,13 @@ export const getISODateTime = (date?: Date | string) => {
...
 
-export const getFormattedDateAndTime = (date: Date) => {
-  return new Date(date).toLocaleDateString("en-us", {
-    weekday: "long",
-    year: "numeric",
-    month: "long",
-    day: "numeric",
+export const getFormattedDateAndTime = (date: Date)=>{
+  return new Date(date).toLocaleDateString('en-us',  { 
+    weekday: 'long', year: 'numeric', month: 'long', day: 'numeric',hour: '2-digit', minute: '2-digit', hour12: true
```
Comment (kurianbenoy-aot) [inline camunda-formio-tasklist-vue/src/services/FormatTime.ts:11]: split it into separate lines for readability

Comment (shuhaib-aot) [reply camunda-formio-tasklist-vue/src/services/FormatTime.ts]: done

Comment (shuhaib-aot) [reply camunda-formio-tasklist-vue/src/services/FormatTime.ts]: this issue fixed

Comment (kurianbenoy-aot) [reply camunda-formio-tasklist-vue/src/services/FormatTime.ts]: Ok

**Review by kurianbenoy-aot — APPROVED** [review]
LGTM 👍

**Pull request merged**